bucket/snipaste-beta.json

```diff
@@ -13,11 +13,9 @@
             "hash": "sha1:5023432a7611f72efd1f6ba84613b61b5eccec62"
         }
     },
-    "pre_install": "if (!(Test-Path \"$persist_dir\\config.ini\")) { Set-Content \"$dir\\config.ini\" '' -Encoding ASCII }",
-    "persist": [
-        "history",
-        "config.ini"
-    ],
+    "pre_install": "if (Test-Path \"$persist_dir\\config.ini\") { Copy-Item \"$persist_dir\\config.ini\" \"$dir\\config.ini\" }",
```

Comment:
You should have left the part that allows the manifest to create the `config.ini` file, if you just have **Copy-Item**, it's not going to work at all.

Reply:
```suggestion
    "pre_install": [
    "if (Test-Path \"$persist_dir\\config.ini\") { Set-Content \"$dir\\config.ini\" '' -Encoding ASCII }",
    "Copy-Item \"$persist_dir\\config.ini\" \"$dir\\config.ini\""
    ],
```

Reply:
```suggestion
    "pre_install": [
    "if (Test-Path \"$persist_dir\\config.ini\") { Set-Content \"$dir\\config.ini\" '' -Encoding ASCII }"
    "Copy-Item \"$persist_dir\\config.ini\" \"$dir\\config.ini\""
    ],
```

Reply:
I tested it, using just **Copy-Item** does not work. You, need to use Set-Content/New-Item to create the `config.ini` file.
![(explorer)73p7R9_11-30-2022](https://user-images.githubusercontent.com/101912712/204953025-1ee6ae55-3dc4-4812-8588-c6a96050e3e5.jpg)

Reply:
oh I got it. Snipaste will create `config.ini` when launched for the first time. So if bare installing then uninstalling without launching it once, there will be an error in `pre_uninstall`.

<details>
<summary>logs</summary>
<pre>
❯ scoop uninstall snipaste-beta -p
Uninstalling 'snipaste-beta' (2.8.1-Beta).
Running pre_uninstall script...
Removing shim 'Snipaste.shim'.
Removing shim 'Snipaste.exe'.
Removing shortcut ~\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Scoop Apps\Snipaste.lnk
Unlinking ~\scoop\apps\snipaste-beta\current
Removing persisted data.
'snipaste-beta' was uninstalled.
</pre>
<pre>
❯ scoop install snipaste-beta
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Installing 'snipaste-beta' (2.8.1-Beta) [64bit] from Versions bucket
Loading Snipaste-2.8.1-Beta-x64.zip from cache.
Checking hash of Snipaste-2.8.1-Beta-x64.zip ... ok.
Extracting Snipaste-2.8.1-Beta-x64.zip ... done.
Running pre_install script...
Linking ~\scoop\apps\snipaste-beta\current => ~\scoop\apps\snipaste-beta\2.8.1-Beta
Creating shim for 'Snipaste'.
Creating shortcut for Snipaste (Snipaste.exe)
Persisting history
'snipaste-beta' (2.8.1-Beta) was installed successfully!
'snipaste-beta' suggests installing 'extras/vcredist2022'.
</pre>
<pre>
❯ scoop uninstall snipaste-beta
Uninstalling 'snipaste-beta' (2.8.1-Beta).
Running pre_uninstall script...
Copy-Item: Cannot find path 'C:\Users\my_username\scoop\apps\snipaste-beta\2.8.1-Beta\config.ini' because it does not exist.
Removing shim 'Snipaste.shim'.
Removing shim 'Snipaste.exe'.
Removing shortcut ~\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Scoop Apps\Snipaste.lnk
Unlinking ~\scoop\apps\snipaste-beta\current
'snipaste-beta' was uninstalled.
</pre>
</details>

I'll fix it. Maybe it's safer to use `Test-Path` in 'pre_uninstall' bc it's closer to this possible error.